frontend/src/app.css

```diff
@@ -13,6 +13,22 @@
     @apply focus:outline-none;
   }
 
+  input[type='range'] {
+    &::-webkit-slider-thumb {
+      @apply bg-green-800 rounded-full appearance-none cursor-pointer;
+    }
+    &::-webkit-slider-runnable-track {
+      @apply h-1.5;
+    }
+
+    &::-moz-range-thumb {
+      @apply bg-green-800 rounded-full appearance-none cursor-pointer;
+    }
+    &::-moz-range-track {
+      @apply h-1.5;
+    }
+  }
```

Comment:
Now much better. Isn't it possible to combine selectors like this: `&::-webkit-slider-thumb, &::-moz-range-thumb` and `&::-webkit-slider-runnable-track, &::-moz-range-track`? Because the styling is identical…

Reply:
I have already tried that but nested/combine with pseudo classes wasn't working that's why I implemented like the above.
please visit the below link
https://stackoverflow.com/questions/50580986/why-cant-i-combine-pseudo-element-selectors

Reply:
```
You are combining multiple vendor-specific selectors into a single CSS rule.

This means that if one of the selectors is not recognised by the browser, the entire CSS block is ignored. In this particular case, Chrome does not recognize ::-moz-range-track, because it is specific to Firefox/Gecko. This is not a quirk, but intended behaviour and [part of the CSS standard](https://www.w3.org/TR/CSS21/syndata.html#rule-sets).
```

Reply:
Oh I see. Thank you very much for a reasonable explanation with a proof. Then keep it as is.